pkgs/applications/misc/zerobin/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv
+, python3Packages
+, pythonOlder
+, buildPythonApplication
+, fetchFromGitHub
+, nodePackages
+, doit
+}:
+
+buildPythonApplication rec {
+  pname = "zerobin";
+  version = "1.0.5";
+  src = fetchFromGitHub {
+    owner = "Tygs";
+    repo = "0bin";
+    rev = "v${version}";
+    sha256 = "sha256-JvgPWkycHJApHZJcvf3Oxm0fq625jr0FJ5/6IRAc3rU";
+  };
+
+  disabled = pythonOlder "3.7";
+
+  nativeBuildInputs = with python3Packages; [ doit pyscss nodePackages.uglify-js ];
+  propagatedBuildInputs = with python3Packages; [ appdirs beaker bleach bottle clize lockfile paste ];
```

Comment:
📝 In my previous review, I [raised one concern](https://github.com/NixOS/nixpkgs/pull/98734#discussion_r495502385) about how zerobin [pins its dependencies to exact versions](https://github.com/Tygs/0bin/blob/v1.0.5/setup.cfg#L15-L22). Since then, I looked around at how strict dependencies are dealt with; it seems like using a patch file to relax those dependencies would be a workaround when it comes to it. 

- Looked for imports done: `git grep buildPythonApplication -- pkgs | cut -d: -f1 | grep -v all-packages | grep -v python-packages.nix | xargs grep -E "import.*\.nix"`
  [nixpart v0.4](https://github.com/NixOS/nixpkgs/tree/27f084d817d6486ab0c2c23aff4305404646c2fa/pkgs/tools/filesystems/nixpart/0.4) seems to be about the only package that uses its own dependency definitions instead of `pythonPackages` / `python3Packages` etc.
- Came across [one package](https://github.com/NixOS/nixpkgs/blob/27f084d817d6486ab0c2c23aff4305404646c2fa/pkgs/servers/home-assistant/relax-dependencies.patch) that uses a patch to relax some dependencies, [and another](https://github.com/NixOS/nixpkgs/blob/27f084d817d6486ab0c2c23aff4305404646c2fa/pkgs/development/python-modules/hwi/relax-deps.patch)

In short, I think this is fine and nothing to do here further.